content/notifications/send-notifications/_index.en.md

```diff
@@ -14,11 +14,61 @@ toc: true
 ## Combining notification channels
 
 {{% notice info %}}
-Per January '24 it is not possible to combine notification channels for a single order request.
+Per April '24 it is not possible to combine notification channels for a single order request.
```

Comment:
```suggestion
As of April '24 it is not possible to combine notification channels for a single order request.
```
Very much nitpicking here 😅, but I think [per](https://dictionary.cambridge.org/dictionary/english/per) has a different meaning in english.